### ISSUE TITLE
The game does not start

### ISSUE THREAD

**Issue by saledarm** [Voxelum/x-minecraft-launcher#736]
![2024-08-20_19-38-07](https://github.com/user-attachments/assets/4cf606fe-bc78-4454-ab70-97e99fd73272)
He writes that there is no Java, although the latest version has been downloaded
![image](https://github.com/user-attachments/assets/5c1d66e0-3208-4a9d-a2a5-c359c943a83f)
What to do?
[report.zip](https://github.com/user-attachments/files/16679029/report.zip)

**Comment by v1mkss** [comment]
Are you using **Labymod**? 
Unfortunately, it's not working at the moment

**Comment by saledarm** [comment]
No, I didn't use it, I just downloaded the launcher and this error came out.

**Comment by v1mkss** [comment]
Ah, 1.19, this version requires java 17 (minimum), I recommend installing version 21 or the same 17, because the new version does not support older versions of java

**Comment by saledarm** [comment]
I downloaded the 22nd version of Java anyway, but the same error still comes out, I don't understand why :(

**Comment by v1mkss** [comment]
Have you selected java 22 in the launcher settings?

**Comment by saledarm** [comment]
And where can I find it in the settings?

**Comment by v1mkss** [comment]
Open your instance, then click here 
![image](https://github.com/user-attachments/assets/32970ffb-4670-4d7f-9a69-b5b1fda5aad0)
Scroll down and find Java, if you haven't changed the installation path, then Java should be here, if not, try reloading (click the reload button) or adding Java 22 
![image](https://github.com/user-attachments/assets/ea61f89c-4f3b-474f-96cc-6c26097d151c)

**Comment by saledarm** [comment]
Everything worked, I just didn't even know about it, thanks a lot :)

**Comment by v1mkss** [comment]
Enjoy the game :) Good luck
